manifests/init.pp

```diff
@@ -243,6 +245,7 @@
     class { 'python':
       version => $python_version,
       dev     => 'present',
+      venv    => 'present',
```

Comment:
Ah! I was just hunting the CI failure and did not realized this is a whole new parameter.  So this is another breaking-change for this module then!  Let me update the PR!

Reply:
So, I will create a new PR containing only the first commit, this will allow to update this PR to detail the reason for the breaking change.  That way the changelog will be easier to follow…

Reply:
Done! #368 has the initial code that I wanted to push, this PR Fix CI and #367 is also remove an unused dependency.

Thank you for the hint @bastelfreak !